packages/app/src/server/routes/api/cortex.ts

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright 2020 Opstrace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import httpProxy from "http-proxy";
+import express from "express";
+import querystring from "querystring";
+
+import { log } from "@opstrace/utils/lib/log";
+
+export const cortexProxy = httpProxy.createProxyServer({ ignorePath: true });
+/**
+ * bodyParser middleware (which we use earlier in chain to parse POST body content), doesn't play
+ * well with the http-proxy, since the body is parsed and altered. It results in POSTs hanging until the connection times out.
+ *
+ * This is a workaround to restream the already parsed body, for the proxy target to consume.
+ */
+cortexProxy.on("proxyReq", function onProxyReq(proxyReq, req, res) {
```

Comment:
We can import {onProxyReq} from "server/utils" now and use that here instead of redeclaring the function again.